test/guest/lib/zero-disk.setup

```diff
@@ -18,9 +18,9 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
-# on Debian we don't want to delete pbuilder
+# on Debian we don't want to delete pbuilder or cracklib
 if [ -d "/var/cache" ]; then
-    ( cd /var/cache; ls . | grep -vx pbuilder | xargs rm -rf )
+    ( cd /var/cache; ls . | grep -vx "pbuilder\|cracklib" | xargs rm -rf )
```

Comment:
> It seems that using 'find' and -prune here would be more maintainable, no?

What's the exact command you recommend to use instead? I found that this seems very readable while not recursing into subdirectories.